### PR TITLE
Wenduo Zhou added

### PR DESCRIPTION
**Pull request by berrakozer**
Work in progress.

### CODE REVIEW

**Comment by berrakozer** [comment]
@sbillinge PR good to go!

File: db/people.yml [hunk 1]
```diff
@@ -2693,3 +2693,84 @@ zzgong:
...
+      organization: University of Oxford
+      position: Royal Society University Research Fellow
+    - begin_year: 2007
+      begin_month: 8
+      end_month: 12
+      end_year: 2011
+      organization: University of Oxford
+      position: Glasstone Fellow
+  name: Asel Sartbaeva
+  position: Lecturer in Chemistry
```
Comment (sbillinge) [inline db/people.yml:2776]: needs a line feed

File: db/people.yml [hunk 2]
```diff
@@ -2693,3 +2693,84 @@ zzgong:
...
+      end_year: 2004
+    - begin_year: 2001
+      degree: MPhil in Mineral Physics
+      institution: University of Cambridge
+      end_year: 2002
+    - begin_year: 1994
+      degree: MSc in Natural Science
+      institution: Kyrgyzskij Rossijskij Slavjanskij Universitet
+      end_year: 1999
+  employment:
```
Comment (sbillinge) [inline db/people.yml:2752]: I think there should be an entry that shows her as a visitor in my group, no?

File: db/people.yml [hunk 3]
```diff
@@ -2693,3 +2693,84 @@ zzgong:
...
+    - begin_year: 2008
+      begin_month: 5
+      end_month: 6
+      end_year: 2009
+      group: bg
+      status: postdoc
+      advisor: sbillinge
+      organization: Columbia University
+      position: Assistant Research Scientist
+    - begin_year: 2006
```
Comment (sbillinge) [inline db/people.yml:2723]: this appointment should also have group: bg and advisor: sbillinge.  He was in  my group at MSU

File: db/people.yml (hunk 2 above)
Comment (berrakozer) [reply db/people.yml:2752]: @sbillinge I saw online that she was your Post Doctoral Associate at Arizona State University. Should I put that for the time period 2005-2007 or she was also in your group in Columbia?

Comment (sbillinge) [reply db/people.yml]: I was at Michigan State, not Arizona State.  She was never at Columbia.  Can you send me what it says online about her history?  Send a link maybe?  Sorry it is complicated...

**Comment by sbillinge** [comment]
Thanks @berrakozer !!!!!

**Pull request merged**